scripts/installDepsScripts/installRustDeps.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+
+# Check if the default Rust comes from rustup, based on the official
+# recommended method of installation:
+# https://www.rust-lang.org/learn/get-started
+declare is_rustup_default=$(rustup show | awk '/\(default\)/')
+
+
+# If Rust comes from rustup, update Rust
+if [[ $is_rustup_default ]]; then
+
+    rustup update
+
+else
+    sleep 1
```

Comment:
Why sleep here?

Reply:
I added this because the output of the previous `declare is_rustup_default=$(rustup show | awk '/\(default\)/')` command, when the command `rustup show` fails as it does not have rustup installed, was messed with the input of `sudo` asking for the password in the next command. So I added this `sleep 1` to make sure the output of the command and the input asking for the passwowrd where nicely shown in sequence in the terminal